### PR TITLE
refactor: Run `lintfix` (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
- Fix autofixable violations
- Remove unused directives
- Allow for PascalCased variables - needed for dynamically imported or assigned classes, decorators, routers, etc.

### CODE REVIEW

File: packages/@n8n_io/eslint-config/base.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ const config = (module.exports = {
 			},
 			{
 				selector: 'variable',
-				format: ['camelCase', 'snake_case', 'UPPER_CASE'],
+				format: ['camelCase', 'snake_case', 'UPPER_CASE', 'PascalCase'],
```
Comment (ivov) [inline packages/@n8n_io/eslint-config/base.js:234]: Or remove this selector entirely?

**Comment by ivov** [comment]
TODO: Add to `.git-blame-ignore-revs` after merge

File: packages/workflow/package.json [hunk 2]
```diff
@@ -29,8 +29,8 @@
     "typecheck": "tsc",
     "build": "tsc -p tsconfig.build.json",
     "format": "prettier --write . --ignore-path ../../.prettierignore",
-    "lint": "eslint . --quiet",
-    "lintfix": "eslint . --fix",
+    "lint": "eslint . --quiet --report-unused-disable-directives",
+    "lintfix": "eslint . --fix --report-unused-disable-directives",
```
Comment (netroy) [inline packages/workflow/package.json:33]: we should revert these

**Review by netroy — CHANGES_REQUESTED** [review]
besides the `package.json` changes, LGTM

File: packages/workflow/package.json (hunk 2 above)
Comment (ivov) [reply packages/workflow/package.json:33]: Please see [69fb7f8](https://github.com/n8n-io/n8n/pull/7537/commits/69fb7f86250587a99901817545cfab89034f7969), I probably realized while you were reviewing

Comment (netroy) [reply packages/workflow/package.json:33]: I mean, you missed to remove the flag from this one `package.json`.

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.15.1`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.15.1)

**Pull request merged**